packages/aws-cdk/lib/init.ts

```diff
@@ -264,6 +264,7 @@ async function initializeGitRepository(workDir: string) {
   print('Initializing a new git repository...');
   try {
     await execute('git', ['init'], { cwd: workDir });
+    await execute('git', ['checkout', '-b', 'main'], { cwd: workDir });
```

Comment:
as of [Git 2.28.0](https://github.blog/2020-07-27-highlights-from-git-2-28/), `default` can be configured using the `git config --global init.defaultBranch main` or whatever a user wants.

Although I think this is what we want if that value isn't set, what about users that have their own property set up there? `i.e. foo` - wouldn't the user expectation be that the default value they assign to be honored?

Reply:
I agree. We shouldn’t be opinionated here.